### PR TITLE
[universe] Display tooltips using LSP

### PR DESCRIPTION
**Pull request by slimsag**
Please see individual commit messages for descriptions.

Will be enabled for all repositories, but only for staging.

### CODE REVIEW

File: services/httpapi/hover_info.go [hunk 1]
```diff
@@ -30,31 +51,68 @@ func serveRepoHoverInfo(w http.ResponseWriter, r *http.Request) error {
...
-	def, err := cl.Defs.Get(ctx, &sourcegraph.DefsGetOp{
-		Def: *defSpec,
-		Opt: &sourcegraph.DefGetOptions{
-			Doc: true,
-		},
-	})
-	if err != nil {
-		return err
-	}
+	if !feature.Features.Universe {
```
Comment (neelance) [inline services/httpapi/hover_info.go]: I'd prefer `if feature.Features.Universe {` here and then an early exit.

File: lang/golang/cmd/langserver-go/Dockerfile [hunk 2]
```diff
@@ -3,4 +3,10 @@ FROM alpine:3.4
 ADD guru /guru
 ADD langserver-go /langserver-go
 
+# HACK: clone the Sourcegraph repository for testing purposes until we figure
```
Comment (neelance) [inline lang/golang/cmd/langserver-go/Dockerfile]: Nice hack (for testing). ;-)

**Comment by neelance** [comment]
LGTM, 1 actionable comment.

File: services/httpapi/hover_info.go (hunk 1 above)
Comment (slimsag) [inline services/httpapi/hover_info.go]: Done.

File: lang/golang/cmd/langserver-go/Dockerfile (hunk 2 above)
Comment (slimsag) [inline lang/golang/cmd/langserver-go/Dockerfile]: Yeah.. also there was some additional confusion here. I had to update it to set GOPATH in `langprocessor-go/Dockerfile` instead (not this dockerfile)

**Comment by slimsag** [comment]
Squashed into logical commits and merging.

**Pull request merged**